examples/runqslower/build.rs

```diff
@@ -1,23 +1,14 @@
-use std::{fs::create_dir_all, path::Path};
-
+use std::{env, path::PathBuf};
 use libbpf_cargo::SkeletonBuilder;
 
-const SRC: &str = "./src/bpf/runqslower.bpf.c";
+const SRC: &str = "src/bpf/runqslower.bpf.c";
 
 fn main() {
-    // It's unfortunate we cannot use `OUT_DIR` to store the generated skeleton.
-    // Reasons are because the generated skeleton contains compiler attributes
-    // that cannot be `include!()`ed via macro. And we cannot use the `#[path = "..."]`
-    // trick either because you cannot yet `concat!(env!("OUT_DIR"), "/skel.rs")` inside
-    // the path attribute either (see https://github.com/rust-lang/rust/pull/83366).
-    //
-    // However, there is hope! When the above feature stabilizes we can clean this
-    // all up.
```

Comment:
Can you clarify why this is now okay to do?

Reply:
The commit message tries to summarize it. The comment says
> ```rs
>     // Reasons are because the generated skeleton contains compiler attributes
>     // that cannot be `include!()`ed via macro. […]
> ```
If you tried that before my commit, you'd get errors about the `#![]`-style attributes. I removed those by moving all of the definitions into a submodule where the attributes can be applied using `#[]`-style attributes.

Reply:
Gotcha, thanks for clarifying